scripts/lib/install

```diff
@@ -451,13 +451,6 @@ fi
 
 "$ZULIP_PATH"/scripts/zulip-puppet-apply -f
 
-if [ "$package_system" = apt ]; then
-    apt-get -y upgrade
-elif [ "$package_system" = yum ]; then
-    # No action is required because `yum update` already does upgrade.
-    :
-fi
```

Comment:
This used to run even if `NO_DIST_UPGRADE` was set, unlike the `apt-get dist-upgrade` above.  Does this affect one-click apps in some way, then?

Reply:
Yeah, it'd be very helpful to have that information. I can't find it; the closest discussion I could find is https://chat.zulip.org/#narrow/stream/3-backend/topic/apt-daily/near/786402.

I wouldn't be surprised if the issue were that at some point `apt-get dist-upgrade` was resulting in problematic prompts for the DO installer mechanism's weird way of running from the first SSH session into the droplet?

Reply:
I have forwarded an email which has more context.